packages/trimerge-sync/src/testLib/MemoryLocalStore.test.ts

```diff
@@ -85,58 +99,6 @@ Array [
       ],
       undefined,
     );
-    expect(fn.mock.calls).toMatchInlineSnapshot(`
-Array [
-  Array [
-    Object {
-      "commits": Array [
-        Object {
-          "metadata": undefined,
-          "ref": "test1",
-        },
-      ],
-      "syncId": "0",
-      "type": "commits",
-    },
-    false,
-  ],
-  Array [
-    Object {
-      "type": "ready",
-    },
-    false,
-  ],
-  Array [
-    Object {
-      "connect": "offline",
-      "read": "offline",
-      "save": "pending",
-      "type": "remote-state",
-    },
-    false,
-  ],
-  Array [
-    Object {
-      "acks": Array [
-        Object {
-          "ref": "test1",
-        },
-      ],
-      "syncId": "1",
-      "type": "ack",
-    },
-    false,
-  ],
-  Array [
-    Object {
-      "connect": "offline",
-      "read": "offline",
-      "save": "saving",
-      "type": "remote-state",
-    },
-    false,
-  ],
-]
-`);
+    expect(fn.mock.calls).toEqual(callsBeforeShutdown);
```

Comment:
does jest mutate fn.mock.calls? you probably want to clone it first. or just compare the length in this case

Reply:
Done!